src/pages/home/HowItWorksSection.tsx

```diff
@@ -110,8 +110,8 @@ export function HowItWorksSection() {
               <p>
                 <InfoCircleOutlined />{' '}
                 <Trans>
-                  Juicebox is new, unaudited, and not guaranteed to work
-                  perfectly. Before spending money, do your own research:{' '}
+                  Juicebox is new and not guaranteed to work perfectly. Before
```

Comment:
can we also remove "new"? maybe something like

> Juicebox is not guaranteed to be free of bugs or exploits. Before spending money, do your own research:

Reply:
Yep great call. Looks like dis now.

<img width="583" alt="Screen Shot 2022-07-30 at 10 47 48 AM" src="https://user-images.githubusercontent.com/12551741/181864812-8003234c-a5f9-4c82-80fd-84595bb15051.png">